test/pkg_RCall.jl

```diff
@@ -0,0 +1,98 @@
+module ValidateRCall_rdiversity
+using Base.Test
+
+using Diversity
+using Diversity.ShortNames
+using RCall
+using DataFrames
+
+# Only run R on macs
+if is_apple()
+    # Environment variable to avoid boring R package builds
+    is_me = haskey(ENV, "SKIP_R_INSTALL") && ENV["SKIP_R_INSTALL"] == "1"
+    # Skip the (slow!) R package installation step
+    if is_me
+        R"library(rdiversity)";
+    else
+        libdir = mktempdir();
```

Comment:
It creates a folder in /tmp, so I was relying on the OS to clean it up. However, I've now fixed this in master.